From 7aa6514f6eedff4342d38d111049719b31e5bbd0 Mon Sep 17 00:00:00 2001
From: Joe Hirschfeld <j@ibj.io>
Date: Sat, 2 Dec 2023 13:41:48 -0800
Subject: [PATCH 2/6] Correct format string for STREAM_PART

The format string previously was using a simple unsigned int while we are using a size_t.
---
 output/http_jpeg.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/output/http_jpeg.c b/output/http_jpeg.c
index aeecffa..126dedc 100644
--- a/output/http_jpeg.c
+++ b/output/http_jpeg.c
@@ -20,7 +20,7 @@ static const char *const STREAM_HEADER = "HTTP/1.0 200 OK\r\n"
                                          "Content-Type: multipart/x-mixed-replace;boundary=" PART_BOUNDARY "\r\n"
                                          "\r\n"
                                          "--" PART_BOUNDARY "\r\n";
-static const char *const STREAM_PART = "Content-Type: " CONTENT_TYPE "\r\n" CONTENT_LENGTH ": %u\r\n\r\n";
+static const char *const STREAM_PART = "Content-Type: " CONTENT_TYPE "\r\n" CONTENT_LENGTH ": %zu\r\n\r\n";
 static const char *const STREAM_BOUNDARY = "\r\n"
                                            "--" PART_BOUNDARY "\r\n";
 
-- 
2.39.3 (Apple Git-145)

